frontend/TestRun/Jenkins/SCTParameterWizard.svelte

```diff
@@ -0,0 +1,414 @@
+                    internalName: "new_scylla_repo",
+                    condition: (params, defs) => (defs.scyllaVersion.currentSource == "rolling_upgrade"),
+                    onChange: function (e, params) {
+                        //empty
+                    }
+                },
+                baseVersions: {
+                    name: "Base Versions to upgrade from",
+                    type: StringParam,
+                    internalName: "base_versions",
+                    condition: (params, defs) => (defs.scyllaVersion.currentSource == "rolling_upgrade"),
+                    onChange: function (e, params) {
+                        //empty
+                    }
+                },
+            }
+        },
+        testConfiguration: {
+            name: "Test configuration",
+            params: {
+                testName: {
+                    name: "Test Name",
+                    type: StringParam,
+                    internalName: "test_name",
+                    condition: (params, defs) => true,
+                    onChange: function (e, params) {
+                        //empty
+                    }
+                },
+                testConfig: {
+                    name: "Configuration to use",
+                    type: StringParam,
+                    internalName: "test_config",
+                    condition: (params, defs) => true,
+                    onChange: function (e, params) {
+                        //empty
+                    }
+                },
+                email_recipients: {
+                    name: "Email Report Recepients",
+                    description: "Comma separated list of email recipients",
+                    type: StringParam,
+                    internalName: "email_recipients",
+                    condition: (params, defs) => true,
+                    onChange: function (e, params) {
+                        //empty
+                    }
+                },
+                stressDuration: {
+                    name: "Custom Stress Command Duration",
+                    type: StringParam,
+                    internalName: "stress_duration",
+                    condition: (params, defs) => true,
+                    onChange: function (e, params) {
+                        //empty
+                    }
+                },
+
+            }
+        },
+        clusterPostTestBehaviour: {
+            name: "Cluster Post-test Behaviour",
+            params: {
+                dbNodes: {
+                    name: "Action for DB Cluster",
+                    type: SelectParam,
+                    internalName: "post_behavior_db_nodes",
+                    condition: (params, defs) => true,
+                    onChange: function (e, params) {
+                        //empty
+                    },
+                    values: ["keep", "keep-on-failure", "destroy"],
+                    labels: ["Preserve", "Preserve on Failure", "Destroy"],
```

Comment:
please don't rename the values like that, it just gonna cause confusion

maybe we want to add some help text for each, that would explain a bit better the behavior, but like this I will tell someone to run with keep, and he won't find it..

Reply:
Done